src/main/java/frc/robot/subsystems/Arm.java

```diff
@@ -0,0 +1,40 @@
+package frc.robot.subsystems;
+
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import com.ctre.phoenix.motorcontrol.can.TalonFX;
+
+public class Arm {
+    public final class Constants{
+        public static final int LEFT_CLIMBER_ID = 1; 
+        public static final int RIGHT_CLIMBER_ID = 2;
+        public static final int CLAW_STRING_ID = 3;
+        //CAN IDs for three motors
+
+        public static final double JOINT_HEIGHT_FROM_GROUND =1;
+        //joint height from ground 
+
+        public static final double ARM_MIN_LENGTH = 0.5;
+        public static final double ARM_MAX_LENGTH = 1;
+        //Arm length limits
+    }
+    public class ClimberSubsystem{
```

Comment:
Since you'll be moving the Constants class, this inner class will be unnecessary. You can put all of the contents of this straight into the outer class.